mantidimaging/core/utility/progress_reporting/progress.py

```diff
@@ -167,7 +167,7 @@ def _format_time(t: SupportsInt) -> str:
         t = int(t)
         return f'{t // 3600:02}:{t % 3600 // 60:02}:{t % 60:02}'
 
-    def update(self, steps: int = 1, msg: str = "", force_continue: bool = False) -> None:
+    def update(self, steps: int = 1, msg: str = "", force_continue: bool = False, cancelable: bool = False) -> None:
```

Comment:
Can this be passed into `start_async_task_view()` from `ReconstructWindowPresenter.do_reconstruct_volume()` instead of `update()`.